tests/components/samsungtv/test_config_flow.py

```diff
@@ -71,9 +71,24 @@ def remote_fixture():
         yield remote
 
 
-async def test_user(hass, remote):
-    """Test starting a flow by user."""
+@pytest.fixture(name="remotews")
+def remotews_fixture():
+    """Patch the samsungtvws SamsungTVWS."""
+    with patch("samsungtvws.SamsungTVWS") as remotews_class, patch(
```

Comment:
I think this path is wrong. We need to patch where the name is looked up.

https://docs.python.org/3/library/unittest.mock.html#where-to-patch

I think it should be this.
```suggestion
    with patch("homeassistant.components.samsungtv.bridge.SamsungTVWS") as remotews_class, patch(
```

Reply:
I recommend speccing the class if possible.

https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch

Reply:
I'll try but honestly I do not understand...
`remotews` fixture (patching the `samsungtvws.SamsungTVWS` class) is exactly the same as the `remote` fixture (that patches the `samsungctl.Remote` class).

Why `remote` works and `remotews` doesn't?!?! Maybe something inside the class that doesn't allow the class to be patched?

Reply:
The other test should be changed too most likely.

Reply:
@MartinHjelmare your suggestion works. Thanks. I used similar name even for the remote fixture.
Curiously enough (at least for me) with my version I got strange behaviour:
* defining the test as test_user(hass, remote, remotews) caused only remote being patched 
* defining the test as test_user(hass, remotews, remote) caused only remotews being patched 
using your "syntax" both got patched.